azure-function-app/massdriver.yaml

```diff
@@ -0,0 +1,188 @@
+schema: draft-07
+name: <md .Name md>
+description: <md .Description md>
+source_url: github.com/YOUR_ORG/<md .Name md>
+access: private
+type: application
+
+app:
+  envs:
+    # Use jq expressions to build environment variables from input params or connections
+    LOG_LEVEL: .params.log_level
```

Comment:
We should have a param ref’d in the example (that’s uncommented) that doesn’t actually exist. It’ll probably fail during deployment t. Saw this in the k8s examples the other day.

Reply:
Sure. Is there an example I can follow? Looked at both k8s example apps and I'm not sure how that would translate here.

Reply:
I’d say add a param as an example w enum “info” “warn” “error” and comment both out. The user then has a working example they can uncomment if it applies.

Reply:
Should this be added to the function app example or the function app template?

Reply:
Template. It does the jq expression eval

Reply:
Something like this?
```
    LOG_LEVEL: # .params.log_level
      # type: string
      # enum:
      #   - info
      #   - warn
      #   - error
```